gateway-core/components/org.wso2.carbon.gateway.core/src/main/java/org/wso2/carbon/gateway/core/config/dsl/external/deployer/IFlowDeployer.java

```diff
@@ -175,7 +177,7 @@ private void updateESBConfig(Artifact artifact) {
             //                ConfigRegistry.getInstance().addGWConfig(configHolder);
             //            }
 
-        } catch (IOException e) {
+        }  catch (IOException e) {
```

Comment:
Would be better if we can remove the extra space. Would be better if we can add check style plugin to avoid this kind of syntax errors